src/store/config.js

```diff
@@ -38,7 +38,7 @@ const mutations = {
     state.server = config.server
     state.auth = config.auth
     state.openIdConnect = config.openIdConnect
-    state.rootFolder = config.rootFolder === undefined ? '/' : config.rootFolder
+    state.rootFolder = config.rootFolder === undefined ? '' : config.rootFolder
```

Comment:
why this change? If I recall correctly we rely on rootfolder being / if not set ....